rules/xcodeproj.bzl

```diff
@@ -282,7 +282,7 @@ def _xcodeproj_impl(ctx):
         "BAZEL_BUILD_EXEC": "$BAZEL_STUBS_DIR/build-wrapper",
         "BAZEL_OUTPUT_PROCESSOR": "$BAZEL_STUBS_DIR/output-processor.rb",
         "BAZEL_PATH": ctx.attr.bazel_path,
-        "BAZEL_RULES_IOS_OPTIONS": "--@build_bazel_rules_ios//rules:local_debug_options_enabled",
+        "BAZEL_RULES_IOS_OPTIONS": "--@build_bazel_rules_ios//rules:local_debug_options_enabled --features=-swift.cacheable_swiftmodules --features=-swift.use_global_module_cache",
```

Comment:
will this cause `bazel run` producing a different path than `bazel build` just like before? (the tests are now not doing anything btw)

Reply:
It was my observation that this does not change the path from what it was before.

Spoke with gyfelton offline about the tests ... they are doing almost the same thing as before, except for they are now looking for framework search paths (instead of header search paths.)